tests/migration/migration.go

```diff
@@ -1352,7 +1386,12 @@ var _ = SIGMigrationDescribe("VM Live Migration", func() {
 				Entry("[test_id:8612] with PVC", func() *v1.VirtualMachineInstance {
 					dv = createDataVolumePVCAndChangeDiskImgPermissions(testsuite.NamespacePrivileged, size)
 					// Use the Underlying PVC
-					return tests.NewRandomVMIWithPVC(dv.Name)
+					return libvmi.New(
+						libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
+						libvmi.WithNetwork(v1.DefaultPodNetwork()),
```

Comment:
Should we make these the default maybe? I see you need to add them in a lot of places.

Reply:
I'm not totally sure about it. In the migration file you see it in a lot of places because we want to create migratable    vmi. I don't know if it is worth it to create a `WithMigrationEnabled`(smth like this) option that wrap the two options. In any case I tend to consider it part of a separate PR, due to the, already big, size of the PR. WDYT?
Thanks

Reply:
Yes, not as part of this PR. But we might should consider adding an option like that?

Reply:
I don't think that this would be necessary. I believe that @AlonaKaplan is working to make the default ("bridge") binding migratable. We should still test with Masquerade because it is so much better than bridge (no stealing IP address from virt-launcher)